### ISSUE TITLE
inviting a player on essentials and them joining it causes it to leave with code 0 and signal null

### ISSUE THREAD

**Issue by Glitchy404** [kolbytn/mindcraft#150]
![image](https://github.com/user-attachments/assets/3be0c57c-2d4b-4261-8c64-c203e365cab1)

**Comment by Glitchy404** [comment]
(im lonewolfx2 btw)

**Comment by merll002** [comment]
Does essential maybe switch from offline to online mode?

**Comment by uukelele** [comment]
Yes, essential probably switches from offline to online mode. Additionally, mods in general aren't officially supported and are a bit risky. I'm afraid there is no way for us to be able to make this work, aside from providing the bot with his own Minecraft account. Or, you could start a server in offline mode and invite both the bot and your friend there.
